Gradle - Metaspace error when adding an extension to a Kotlin project

We have the following failure from time to time in the CI:

```
2023-03-04T00:09:06.5633394Z [INFO] Running io.quarkus.gradle.AddExtensionToModuleInMultiModuleKtsProjectTest
2023-03-04T00:09:06.6949624Z $ /home/runner/work/quarkus/quarkus/integration-tests/gradle/./gradlew -DquarkusRegistryClient=true -Dquarkus.tools.config=/home/runner/work/quarkus/quarkus/integration-tests/gradle/build/.quarkus/config.yaml -Dmaven.repo.local=/home/runner/.m2/repository -Dorg.gradle.console=plain --stacktrace --info :application:addExtension --extensions=openshift
2023-03-04T00:09:46.0948634Z Command: /home/runner/work/quarkus/quarkus/integration-tests/gradle/./gradlew -DquarkusRegistryClient=true -Dquarkus.tools.config=/home/runner/work/quarkus/quarkus/integration-tests/gradle/build/.quarkus/config.yaml -Dmaven.repo.local=/home/runner/.m2/repository -Dorg.gradle.console=plain --stacktrace --info :application:addExtension --extensions=openshift ran from: /home/runner/work/quarkus/quarkus/integration-tests/gradle/target/classes/add-extension-multi-module-kts failed with exit code: 1 and the following output:
2023-03-04T00:09:46.0950013Z 
2023-03-04T00:09:46.0950227Z FAILURE: Build failed with an exception.
2023-03-04T00:09:46.0950463Z 
2023-03-04T00:09:46.0950621Z * What went wrong:
2023-03-04T00:09:46.0951055Z A problem occurred configuring project ':application'.
2023-03-04T00:09:46.0951495Z > Could not create task ':application:addExtension'.
2023-03-04T00:09:46.0951952Z    > Could not create task of type 'QuarkusAddExtension'.
2023-03-04T00:09:46.0952390Z       > Could not generate a decorated class for type QuarkusAddExtension.
2023-03-04T00:09:46.0952737Z          > Metaspace
2023-03-04T00:09:46.0952943Z 
2023-03-04T00:09:46.0953085Z * Try:
2023-03-04T00:09:46.0953448Z > Run with --debug option to get more log output.
2023-03-04T00:09:46.0953854Z > Run with --scan to get full insights.
2023-03-04T00:09:46.0954090Z 
2023-03-04T00:09:46.0954230Z * Exception is:
2023-03-04T00:09:46.0954774Z org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':application'.
```

I'm surprised that just adding an exception could lead to the metaspace being full so I wonder if there's something fishy here.

--- a/integration-tests/gradle/src/test/java/io/quarkus/gradle/QuarkusGradleWrapperTestBase.java
+++ b/integration-tests/gradle/src/test/java/io/quarkus/gradle/QuarkusGradleWrapperTestBase.java
@@ -57,6 +57,7 @@ public class QuarkusGradleWrapperTestBase extends QuarkusGradleTestBase {
         command.add(getGradleWrapperCommand());
         command.addAll(getSytemProperties());
         command.add("-Dorg.gradle.console=plain");
+        command.add("-Dorg.gradle.daemon=false");
         command.add("--stacktrace");
         command.add("--info");
         command.addAll(Arrays.asList(args));

